src/c/CMakeLists.txt

```diff
@@ -14,14 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.6)
+cmake_minimum_required(VERSION 3.5)
 
 project(zookeeper VERSION 3.5.3)
```

Comment:
@andschwa - please note, this is a separate issue from this patch, but something we need to address. Notice that the version is incorrect here - it should be 3.6.0 for master. 

Which brings up another issue - version updates in our source files is currently a manual process, see "update the version" references here: https://cwiki.apache.org/confluence/display/ZOOKEEPER/HowToRelease+using+git I believe we will need to add cmake to this list as well.

I also see that at some point the version has not been updated properly. iirc all of the versions should be the upcoming release, not the last one.

Andrew would you mind creating a JIRA/patch to address all of these? Alternately the cmake at a minimum. If anyone has ideas how to do a better job on this perhaps we should consider that as well.

Reply:
Sorry, forgot about this. I'll at least get a patch out that fixes master.

Reply:
https://github.com/apache/zookeeper/pull/484

Reply:
@phunt If you could give the `andschwa` Confluence user edit permissions for the ZooKeeper wiki, I can add to that page how to update the CMake version when releasing.